### PR TITLE
Update web-development.md

### PR DESCRIPTION
**Pull request by pimbrouwers**
## Summary

Adds Falco (framework) to the F# web development document.

NuGet: https://www.nuget.org/packages/Falco
GitHub: https://github.com/pimbrouwers/Falco

## Notes

Falco has grown to the point now where I believe it warrants mention as a reputable option for web programming with F#. It is already listed on the FSharp.org website (https://fsharp.org/guides/web/#falco).

@cartermp, @dsyme was hoping you guys could give the thumbs up to the efficacy, quality and community support.

### CODE REVIEW

**Review by cartermp — APPROVED** [review]

**Comment by pimbrouwers** [comment]
Thanks Phillip! 🥳

File: docs/fsharp/scenarios/web-development.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Other web development options are documented in the F# Software Foundation's [Gu
 
 [WebSharper](https://github.com/dotnet-websharper/core) is a community-driven, full-stack, functional reactive web programming technology for .NET, allowing you to develop microservices, client-server web applications, reactive SPAs, and more in F#.
 
+## Falco 
+
+[Falco](https://github.com/pimbrouwers/Falco) is a community-driven toolkit for building _fast_, functional-first and fault-tolerant web applications using F#. It is built upon the high-performance components of ASP.NET Core, optimized for building HTTP applications quickly, has a built-in view engine, and seamlessly integrates with existing .NET Core middleware and frameworks.
```
Comment (gewarren) [inline docs/fsharp/scenarios/web-development.md:38]:
```suggestion
[Falco](https://github.com/pimbrouwers/Falco) is a community-driven toolkit for building _fast_, functional-first, and fault-tolerant web applications using F#. It's built upon the high-performance components of ASP.NET Core and is optimized for building HTTP applications quickly. Falco has a built-in view engine and seamlessly integrates with existing .NET Core middleware and frameworks.
```

Comment (pimbrouwers) [reply docs/fsharp/scenarios/web-development.md:38]: Great edits! Thank you!

File: docs/fsharp/scenarios/web-development.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ Other web development options are documented in the F# Software Foundation's [Gu
 
 [WebSharper](https://github.com/dotnet-websharper/core) is a community-driven, full-stack, functional reactive web programming technology for .NET, allowing you to develop microservices, client-server web applications, reactive SPAs, and more in F#.
 
+## Falco
```
Comment (gewarren) [inline docs/fsharp/scenarios/web-development.md:36]:
```suggestion
## Falco
```

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**